AOBJ/type/zif_aff_aobj_v1.intf.abap

```diff
@@ -0,0 +1,74 @@
+INTERFACE zif_aff_aobj_v1
+
+  PUBLIC.
+
+  TYPES: ty_write_class TYPE c LENGTH 40.
+  TYPES: ty_delete_class TYPE c LENGTH 40.
+  TYPES: ty_reload_class TYPE c LENGTH 40.
+
+
+  TYPES:
+    BEGIN OF ty_attributes,
+      "! <p class="shorttext">Name of executable class which writes the data to the archive</p>
```

Comment:
These title here are quite long. Since the titles are used as label in the editor, shortened titles would be better.
I took a look in you AOBJ transaction. There you use `Write Program`, `Delete Program`, `Reload Program`. Is there a reason you don't take these as title?

Reply:
We changed the Titles